mcs/build/profiles/testing_aot_full_interp.make

```diff
@@ -0,0 +1,8 @@
+#! -*- makefile -*-
+
+include $(topdir)/build/profiles/testing_aot_common.make
+
+PROFILE_MCS_FLAGS += \
+	-d:DISABLE_COM
```

Comment:
@marek-safar I kept remoting enabled since it passes some tests and I don't think it would hurt to see if it works. We can disable later if there are problems.